app/api/reporting.py

```diff
@@ -26,9 +29,13 @@ def reporting_details(collection_instrument_type, collex_id):
         abort(404, 'Invalid CI type')
 
     try:
-        report = get_reporting_details(collection_instrument_type, collex_id)
+        report = get_reporting_details(survey_id, collex_id)
     except HTTPError:
         logger.debug('Invalid Collection exercise id')
         abort(404)
 
-    return report
+    if collection_instrument_type == 'seft':
```

Comment:
Why aren't we doing the same for eQ?

Reply:
The figures the reporting service returns are already all in the format we use for EQ, it's only SEFT that needs any further calculation done at the moment.